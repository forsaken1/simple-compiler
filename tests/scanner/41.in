'a''b''c''d''\n'