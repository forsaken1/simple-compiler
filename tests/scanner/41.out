1	1	char		a
1	4	char		b
1	7	char		c
1	10	char		d
1	13	escape		\n
1	17	eof		End of file